src/components/CryptoCards/CryptoCards.styles.tsx

```diff
@@ -0,0 +1,177 @@
+import resetCSS from '../../styles/reset';
+import fonts from '../../styles/fonts';
+import color from '../../styles/colors';
+import { css } from 'styled-components';
+import { getShade } from '../../styles/colors';
+
+const borderStyle = css`
+    ${resetCSS}
+    height: 336px;
+    width: 265px;
```

Comment:
Do not use fixed sizes for height/width

Reply:
prop you want width 100%, max with 256, display block.... in this case you maybe do want a set height as they must all stand in a row and we dont want one bigger than the rest... however we could use flex on the parent element for these. fixed height is ok for now

Reply:
done